### PR TITLE
[MIG][10.0-Issue175-T23291] project_timesheet_task_id

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
Migrated project_timesheet_task_id in v10.
@elicoidal @seb-elico @victormartinelicocorp Please review.
This PR refers to issue #175

### CODE REVIEW

**Comment by sudhir-serpentcs** [comment]
Now, this module just add the task_id field in analytic line in timesheet_sheet.sheet object view.
@victormartinelicocorp @elicoidal @seb-elico

File: project_timesheet_task_id/README.rst [hunk 1]
```diff
@@ -41,20 +43,10 @@ Maintainer
...
-* Ready-to-use ERP packages aimed at starting businesses.
-* Odoo implementation for manufacturing, international trading, service industry
-  and e-commerce. 
-* Connectors and integration with 3rd party software (Magento, Taobao, Coswin,
-  Joomla, Prestashop, Tradevine etc...).
-* Odoo Support services such as developments, training, maintenance and hosting.
-
-Our headquarters are located in Shanghai with branch in Singapore servicing
-customers from all over Asia Pacific.
+Elico Corporation offers consulting services to implement open source management
```
Comment (elicoidal) [inline project_timesheet_task_id/README.rst]: I think I miss some content

**Review by elicoidal — APPROVED** [review]
LGTM
@seb-elico any news on the runbot?

**Comment by iceyxu** [comment]
@Elico-Corp/odoo-reviewers  what status for this PR?

**Comment by elicoidal** [comment]
@iceyxu We need to tests all these features but runbot is not yet ready. 
I would suggest that @victormartinelicocorp prepare the trunk environment for Elico with latest modifications and that you test them

**Comment by lonelysun** [comment]
@elicoidal @seb-elico @victormartinelicocorp 
 what status for this PR?

**Comment by victor-ditay** [comment]
>@iceyxu We need to tests all these features but runbot is not yet ready.
I would suggest that @victormartinelicocorp prepare the trunk environment for Elico with latest modifications and that you test them

@lonelysun Never tested AFAIK could you deploy to Elico 10 test environment if we have?

cc @elicoidal @seb-elico

**Comment by elicoidal** [comment]
@victormartinelicocorp Better Runbot

**Comment by victor-ditay** [comment]
>LGTM
@seb-elico any news on the runbot?
    @iceyxu We need to tests all these features but runbot is not yet ready.
    I would suggest that @victormartinelicocorp prepare the trunk environment for Elico with latest modifications and that you test them
@lonelysun Never tested AFAIK could you deploy to Elico 10 test environment if we have?
cc @elicoidal @seb-elico
@victormartinelicocorp Better Runbot


Noted @elicoidal, 
@seb-elico should we create a runbot configuration now?

**Comment by seb-elico** [comment]
@victormartinelicocorp Sounds like a great idea :+1:

**Comment by seb-elico** [comment]
@victormartinelicocorp Sounds like a great idea +1

**Pull request merged**